### DISCUSSION TITLE
Full platform support?

### DISCUSSION THREAD

**Discussion by zxffffffff** [godotjs/GodotJS#6] [Q&A]
Is this project considering supporting mobile in the future?

I found similar schemes in other projects
- [Geequlim/ECMAScript](https://github.com/Geequlim/ECMAScript.git) use quickjs
- [Tencent/puerts](https://github.com/Tencent/puerts.git) use v8, quickjs, nodejs

**Comment by ialex32x** [comment]
Currently, it's just that there isn't much energy to cover more platforms; theoretically, all that's needed is to complete the compilation of `V8` to use it on mobile platforms (and `lws` if remote debugger is needed on mobile). I'll try the arm64 build on Android at least this month.

For the Web build, quickjs is assumed a suitable solution currently available for use in wasm. So it will be implemented in the same or a very similar way `puerts` used.

**Comment by ialex32x** [comment]
More platforms are supported now, but not tested yet :)
